src/alto-ui/Form/DatePicker/DatePicker.js

```diff
@@ -91,9 +113,19 @@ class DatePicker extends React.Component {
     this.setState(({ month }) => ({ month: month.set(obj) }));
   }
 
+  handleChangeInput(e) {
+    this.setState({ manualInputValue: e.target.value });
+  }
+
   formatTextfieldDate(value) {
     // const { open } = this.state; // Uncomment when we enhance for typing in datefield
     const { format, displayFormat } = this.props;
+    const { manualInputValue } = this.state;
+
+    if (manualInputValue) {
+      return manualInputValue;
+    }
```

Comment:
wrong place. `formatTextfieldDate` is the name of the function. You are not formating anything here. Put this logic directly in render.

Reply:
fixed